home_work_6.2.js

```diff
@@ -0,0 +1,13 @@
+const competitorPizzas = ['Dodo', '123', 'DIABLO', 'Papa Johns', 'hawai', 'pepperoni'].map(v => v.toLowerCase());
+const myPizza = ['Dodo', '123', 'aDIABLO', 'Papa Johns', 'hawai', 'pepperoni'].map(v => v.toLowerCase());
+let pizzaBoolean = false;
+for(const onePizza of myPizza){
+    if (!competitorPizzas.includes(onePizza)){
+        console.log(onePizza)
+        pizzaBoolean = true
+    }
+}
+if (pizzaBoolean == false) {
+    console.log(null)
+}
```

Comment:
В условии задания нужно вьівести все пиццьі, если есть 
Создать пустой массив и в него добавлять, `pizzaBoolean` не понадобится
```suggestion
notInCompetitorList.length ? notInCompetitorList : null
```